Content.Server/Power/Components/PowerChargeComponent.cs

```diff
@@ -20,12 +20,20 @@ public sealed partial class PowerChargeComponent : SharedPowerChargeComponent
     [DataField("idlePower")]
     public float IdlePowerUse { get; set; }
 
+    // Frontier: different power when charging vs. charged
     /// <summary>
-    /// Power consumed when <see cref="SwitchedOn"/> is true.
+    /// Power consumed when <see cref="SwitchedOn"/> is true and the power is fully charged.
     /// </summary>
     [DataField("activePower")]
     public float ActivePowerUse { get; set; }
 
+    /// <summary>
+    /// Power consumed when <see cref="SwitchedOn"/> is true and the machine is not fully charged.
+    /// </summary>
+    [DataField("activeChargingPower")]
```

Comment:
```suggestion
    [DataField("activeChargingPower", required: true)]
```
Should probably set this to required or set it to negative and have it set to ActivePowerUse if unset.